lib/calculator.rb

```diff
@@ -0,0 +1,53 @@
+class Calculator
+
+  attr_reader :total_sum, :dishes
+
+  def send_sms
+    answer = gets.chomp
+    if answer == "n"
+      puts "Please order again."
+      Menu.new.view_dishes
+  #  else
+#######  TWILIO HERE ###########
+    end
+  end
+
+  def error_check
```

Comment:
Could this method name be more readable? Perhaps have it specify what sort of errors it checks for, and also what happens if there's an error?